tests/e2e/framework/util.go

```diff
@@ -124,10 +121,9 @@ func CreateTestingNS(ctx context.Context, baseName string, c clientset.Interface
 		if err != nil {
 			if apierrors.IsAlreadyExists(err) {
 				// regenerate on conflict
-				e2elog.Logf("Namespace name %q was already taken, generate a new name and retry", namespaceObj.Name)
 				namespaceObj.Name = fmt.Sprintf("%v-%v", baseName, RandomSuffix())
 			} else {
-				e2elog.Logf("Unexpected error while creating namespace: %w", err)
+				gomega.Expect(err).NotTo(gomega.HaveOccurred(), "Failed to create namespace %s", namespaceObj.Name)
```

Comment:
I would prefer to remove the indentation and then `return false, nil` if already exists.

Note sure about the value of this expect here ... given that we don't do anything else with the errors below.